doc/python/configuration-options.md

```diff
@@ -33,40 +33,150 @@ jupyter:
 
 ##### Enable Scroll Zoom
 
+This option allows users to zoom in and out of figures using the scroll wheel on their mouse and/or a two-finger scroll. 
+
 ```python
 import plotly.graph_objects as go
 
 fig = go.Figure()
 
+config = dict({'scrollZoom': True})
+
 fig.add_trace(
     go.Scatter(
         x=[1, 2, 3],
         y=[1, 3, 1]))
 
-fig.show(config={'scrollZoom': True})
+fig.show(config=config)
 ```
 
-##### Display ModeBar
+##### Force The Modebar to Always Be Visible
+
+When users hover over a figure generated with plotly.py, a modebar appears in the top-right of the figure. This presents users with several options for interacting with the figure.
+
+By default, the modebar is only visible while the user is hovering over the chart. If you would like the modebar to always be visible regardless of whether or not the user is currently hovering over the figure, set the displayModeBar attribute in the configuration of your figure to true.
 
 ```python
 import plotly.graph_objects as go
 
 fig = go.Figure()
 
+config = {'displayModeBar': True}
+
 fig.add_trace(
     go.Scatter(
         x=[1, 2, 3],
         y=[1, 3, 1]))
 
-fig.show(config={'displayModeBar': True})
+fig.show(config=config)
 ```
 
-##### Edit Mode - change the title and axis titles
+##### Never Display The Modebar
+
+When users hover over a figure generated with `plotly.py`, a modebar appears in the top-right of the figure. This presents users with several options for interacting with the figure.
+
+By default, the modebar is only visible while the user is hovering over the chart. If you would like the modebar to never be visible, then set the displayModeBar attribute in the config of your figure to false.
+
+```python
+import plotly.graph_objects as go
+
+fig = go.Figure()
+
+config = {'displayModeBar': False}
+
+fig.add_trace(
+    go.Scatter(
+        x=[1, 2, 3],
+        y=[1, 3, 1]))
+
+fig.show(config=config)
+```
+
+##### Display the `Edit Chart` Link
+
+Set `showLink` to `True` in order to make your figure editable on [Chart Studio](https://plot.ly/online-chart-maker). 
+
+```python
+import plotly.graph_objects as go
+
+fig = go.Figure()
+
+config = {'showLink': True}
+
+fig.add_trace(
+    go.Scatter(
+        x=[1, 2, 3],
+        y=[1, 3, 1]))
+
+fig.show(config=config)
+```
+
+### Display The `Edit In Chart Studio` Modebar Button
+
+```python
+import plotly.graph_objects as go
+
+fig = go.Figure()
+
+config = {'showEditInChartStudio': True}
+
+fig.add_trace(
+    go.Scatter(
+        x=[1, 2, 3],
+        y=[1, 3, 1]))
+
+fig.show(config=config)
+```
+
+##### Hide the Plotly Logo on the Modebar
+
+```python
+import plotly.graph_objects as go
+
+fig = go.Figure()
+
+config = {'displaylogo': False}
+
+fig.add_trace(
+    go.Scatter(
+        x=[1, 2, 3],
+        y=[1, 3, 1]))
+
+fig.show(config=config)
+```
+
+##### Making A Responsive Chart
```

Comment:
This one would deserve more explanations I think: define responsiveness, and comment on figure behaviour when it's responsive. For example when I tried this config parameter does not make a figure responsive (resized when winodw is resized) in a Jupyter notebook, but in a Dash app the figure is responsive by default?

Reply:
Looking at this example, I think it actually makes more sense to make it about how to disable the default responsive behavior that figures have. 

I'm surprised figures aren't responsive in your Jupyter notebook- they are in mine 🤔